### ISSUE TITLE
Run in browser

### ISSUE THREAD

**Issue by pannous** [MaxGraey/Assembleash#6]
please make it that the main function gets executed automatically and the result been show in the browser
```js
export function main():int{
    return fib(42);
}
```

**Comment by MaxGraey** [comment]
Yes, I will plan to implement this, as well as saving and import gists from github. But unfortunately I'm pretty busy right now on other commercial projects.
